website/docs/guides/orchestration/custom-cicd-pipelines/3-dbt-cloud-job-on-merge.md

```diff
@@ -58,6 +58,26 @@ Here’s a video showing these steps:
 
 <WistiaVideo id="u7mo30puql" />
 </TabItem>
+<TabItem value="ADO">
```

Comment:
adding ado as a value so it appears as tabs. currently ado not appearing in deploy preview 
![Uploading Screenshot 2023-06-12 at 10.08.28.png…]()